js/controller.js

```diff
@@ -1638,7 +1638,17 @@ OO.plugin('Html5Skin', function(OO, _, $, W) {
       var uiLanguage = Utils.getLanguageToUse(this.state.config);
       this.mb.publish(OO.EVENTS.SKIN_UI_LANGUAGE, uiLanguage);
 
-      this.state.audioOnly = this.state.config.audio.audioOnly;
+      // Check for valid player type, otherwise default to video
+      switch (params.playerType) {
+        case OO.CONSTANTS.PLAYER_TYPE.AUDIO:
+          this.state.audioOnly = true;
+          break;
+        case OO.CONSTANTS.PLAYER_TYPE.VIDEO:
+        default:
+          this.state.audioOnly = false;
+          break;
+      }
+
```

Comment:
Could we replace it by one line?
`this.state.audioOnly = params.playerType === OO.CONSTANTS.PLAYER_TYPE.AUDIO;`

Reply:
Updated